charts/chronograf/README.md

```diff
@@ -84,35 +78,37 @@ The following table lists the configurable parameters of the chronograf chart an
 | `tolerations`                | Toleration labels for pod assignment                                                                      | []                                          |
 | `affinity`                   | Affinity settings for pod assignment                                                                      | {}                                          |
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+To configure the chart, do either of the following:
 
-```bash
-helm upgrade --install my-release \
-  --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
-    influxdata/chronograf
-```
+- Specify each parameter using the `--set key=value[,key=value]` argument to `helm upgrade --install`. For example, use the following command:
 
-The above command enables persistence and changes the size of the requested data volume to 200GB.
+  ```bash
+  helm upgrade --install my-release \
+    --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
+      influxdata/chronograf
+  ```
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+- Provide a YAML file that specifies parameter values while installing the chart. For example, use the following command:
 
-```bash
-helm upgrade --install my-release -f values.yaml influxdata/chronograf
-```
+  ```bash
+  helm upgrade --install my-release -f values.yaml influxdata/chronograf
+  ```
 
-> **Tip**: You can use the default [values.yaml](values.yaml)
+  > **Tip**: Use the default [values.yaml](values.yaml).
+
+For information about running Chronograf in Docker, see the [full image documentation](https://quay.io/influxdb/chronograf).
 
 ## Persistence
 
 The [Chronograf](https://quay.io/influxdb/chronograf) image stores data in the `/var/lib/chronograf` directory in the container.
 
 The chart optionally mounts a [Persistent Volume](http://kubernetes.io/docs/user-guide/persistent-volumes/) at this location. The volume is created using dynamic volume provisioning.
 
-## OAuth Using Kubernetes Secret
+## OAuth using Kubernetes Secret
```

Comment:
Does it make sense to include this "OAuth using Kubernetes secrets” section for any of our Helm charts other than Chronograf?

Reply:
No, only Chronograf